va/drm/va_drm_utils.c

```diff
@@ -48,14 +48,39 @@ static const struct driver_name_map g_driver_name_map[] = {
     { NULL,         0, NULL }
 };
 
+/* Returns the VA driver candidate num for the active display*/
+VAStatus
+VA_DRM_GetCandidateNum(VADriverContextP ctx, int * candidate_num)
+{
+    struct drm_state * const drm_state = ctx->drm_state;
+    drmVersionPtr drm_version;
+    int num_of_candidate = 0;
+    const struct driver_name_map *m = NULL;
+    if (!drm_state || drm_state->fd < 0)
+        return VA_STATUS_ERROR_INVALID_DISPLAY;
+    drm_version = drmGetVersion(drm_state->fd);
+    if (!drm_version)
+        return VA_STATUS_ERROR_UNKNOWN;
+    for (m = g_driver_name_map; m->key != NULL; m++) {
+        if (drm_version->name_len >= m->key_len &&
+            strncmp(drm_version->name, m->key, m->key_len) == 0)
+        {
```

Comment:
Please don't put the curly braces on a separate line to  keep coding style consistent within the existent code, and update the same coding style issue below.